eclipsePlugin/src/de/tobject/findbugs/FindbugsPlugin.java

```diff
@@ -421,14 +421,14 @@ public static FindbugsPlugin getDefault() {
      */
     public static IWorkbenchWindow getActiveWorkbenchWindow() {
         if (Display.getCurrent() != null) {
-            return getDefault().getWorkbench().getActiveWorkbenchWindow();
+            return PlatformUI.getWorkbench().getActiveWorkbenchWindow();
```

Comment:
`AbstractUIPlugin.getWorkbench()` is deprecated, and simply calls `PlatformUI.getWorkbench()`.